sdk/identity/azure-identity/samples/azure-aad-auth-with-redis.py

```diff
@@ -46,6 +47,12 @@ def re_authentication():
     max_retry = 3
     for index in range(max_retry):
         try:
+            if _need_refreshing(token):
```

Comment:
For the async scheduled logic in the gist, 
we can leave a comment here for users looking to use it that way, can reach out to us via GitHub and we can provide them the guidance/pointers.